tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+"""
+Common test configuration both unit and integration tests.
+"""
+
+from datetime import datetime
+
+import pytest
+
+
+class Whatever:
```

Comment:
Can we add some documentation here for how this class works? The end result seems useful, but I'm not quite sure how it works at first glance. Also, perhaps it deserves a more descriptive name?

Reply:
I added a docstring. I took the name from the Normandy source code. I'm happy to rename if you have suggestions!

Reply:
What about something with the word "constrain", maybe `Constrainer` or `ConstrainedAssertion`?

Reply:
Oh that's interesting. I see it as the complete opposite, like *loose assertion*, *sloppy assertion*

Reply:
As we ended up discussion on Slack, we landed on something to do with `Fuzzy`, like `FuzzyAssertion`.